cdk-project/lib/images/codebuild-image/python/src/notebooks/cli/run_all_notebooks.py

```diff
@@ -21,6 +21,13 @@ def parse_args(args):
         type=bool,
         required=False,
     )
+    parser.add_argument(
+        "--skip-localmode",
```

Comment:
I wanted to check if there is an impact if this is invoked with a false option? If yes then do we need this argument.

Reply:
Yes, in that case it would include those notebooks which actually would be a wrong expectation, these can be removed.